### PR TITLE
Magazines service implementation

### PR DESCRIPTION
**Pull request by NiazBikbaev**

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/exception/IllegalServiceArgumentException.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.epam.magazinestore.exception;
+
+public class IllegalServiceArgumentException extends ServiceException {
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/exception/IllegalServiceArgumentException.java]: Why did you write so many exceptions? Maybe, is it enough to have only one ?

Comment (NiazBikbaev) [reply src/main/java/com/epam/magazinestore/exception/IllegalServiceArgumentException.java]: In order to distinguish different exceptions thrown from dao layer. 
Should I leave only ServiceException?

Comment (AntonKiselyov) [reply src/main/java/com/epam/magazinestore/exception/IllegalServiceArgumentException.java]: You can write only one exception : ApplicationException (as your application names - think how to name it), and write different messages. It's unnecessary to create exception for every layer in your application

Comment (NiazBikbaev) [reply src/main/java/com/epam/magazinestore/exception/IllegalServiceArgumentException.java]: Fixed in 9a03984

**Pull request merged**